codersdk/deployment.go

```diff
@@ -160,6 +161,7 @@ type DeploymentValues struct {
 	DisablePasswordAuth             clibase.Bool                    `json:"disable_password_auth,omitempty" typescript:",notnull"`
 	Support                         SupportConfig                   `json:"support,omitempty" typescript:",notnull"`
 	GitAuthProviders                clibase.Struct[[]GitAuthConfig] `json:"git_auth,omitempty" typescript:",notnull"`
+	CLISSH                          CLISSHConfig                    `json:"cli_ssh,omitempty" typescript:",notnull"`
```

Comment:
Let's call this `SSHConfig` like we are other places

Reply:
Yup. Doing all the renames